examples/basic/main.tf

```diff
@@ -38,12 +38,20 @@ module "ocean-aks-np" {
   tags                                     = { "tagKey": "env", "tagValue": "staging" }
   labels                                   = { "key": "env","value": "test"}
   taints                                   = [{"key":"taintKey","value":"taintValue", "effect" : "NoSchedule"}]
-  vmsizes_filters_min_vcpu                 = 2
-  vmsizes_filters_max_vcpu                 = 16
-  vmsizes_filters_min_memory_gib           = 10
-  vmsizes_filters_max_memory_gib           = 18
-  vmsizes_filters_series                   = ["D v3", "Dds_v4", "Dsv2"]
-  vmsizes_filters_architectures            = ["X86_64"]
+  vmsizes_filters_min_vcpu                 = 4
+  vmsizes_filters_max_vcpu                 = 96
+  vmsizes_filters_min_memory_gib           = 14.0
+  vmsizes_filters_max_memory_gib           = 900.0
+  vmsizes_filters_series                   = ["NC","NV"]
```

Comment:
keep the old value for series ["D v3", "Dds_v4", "Dsv2"]

Reply:
done